dydx/dydxViews/dydxViews/_v4/MarketInfo/Components/Resources/dydxMarketResourcesView.swift

```diff
@@ -61,11 +61,11 @@ public class dydxMarketResourcesViewModel: PlatformViewModel {
                         HStack {
                             VStack(alignment: .leading) {
                                 HStack(spacing: 4) {
-                                    Text(sharedMarketViewModel.tokenFullName ?? "")
+                                    Text(sharedMarketViewModel.assetName ?? "")
                                         .themeColor(foreground: .textPrimary)
                                         .themeFont(fontType: .plus, fontSize: .medium)
                                 }
-                                Text(sharedMarketViewModel.tokenSymbol ?? "")
+                                Text(sharedMarketViewModel.assetId ?? "")
```

Comment:
confirming this is display id?

Reply:
see [comment](https://github.com/dydxprotocol/v4-native-ios/pull/229/files#r1722316474)